src/server_manager/web_app/cloud_accounts.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Account} from "../model/digitalocean";
+
+type DigitalOceanAccountFactory = (accessToken: string) => Account;
+
+export class CloudAccounts {
+  private readonly DIGITALOCEAN_TOKEN_STORAGE_KEY = 'LastDOToken';
```

Comment:
Not for this PR, but we will probably need to change persistence to hold (cloud_id, account_id, token). Perhaps name too. This way we could show the accounts even if the token expires.
- cloud_id is to know which constructor to use
- account_id is to know the accounts that have been connected and give an identity to the token
- token to access the api
- Perhaps name so we can show the email even if the token expires.

Reply:
Sounds like a good idea. I jotted down a note and we can tackle it in another PR.